editor/src/components/text-editor/text-editor.tsx

```diff
@@ -49,6 +62,68 @@ const handleShortcut = (
   ]
 }
 
+const handleSetFontSizeShortcut = (
+  event: React.KeyboardEvent<Element>,
+  metadata: ElementInstanceMetadataMap,
+  elementPath: ElementPath,
+): Array<ApplyCommandsAction> => {
+  const modifiers = Modifier.modifiersForEvent(event)
+  const character = keyCharacterFromCode(event.keyCode)
+  const matches = isAdjustFontSizeShortcut(modifiers, character)
+
+  if (!matches) {
+    return []
+  }
+
+  const direction: { [key: string]: number } = { comma: -1, period: 1 }
```

Comment:
This use of an object is adding a potentially unnecessary `undefined` into the mix later on. Please can you replace this with explicit checks for either of those characters, and return `[]` if we have somehow entered this code for any other keyboard character

Reply:
fixed in https://github.com/concrete-utopia/utopia/pull/3077/commits/ec9f379a406f74b0812f5a94524778619f2a035e